src/templates/interest/html.mjml

```diff
@@ -0,0 +1,78 @@
+<mjml owa="desktop" version="4.6.3">
+  <mj-head>
+    <mj-title>The {{ poolYear }} Confidence Pool is Now Open
+    </mj-title>
+    <mj-preview>{{{formatPreview 'The {{ poolYear }} confidence pool is open and ready for registration.'}}}</mj-preview>
+    {{> styles }}
+    <mj-style>
+      .subtitle {
+        font-weight: bold;
+        font-size: 19px;
+        margin-bottom: 0px;
+        margin-top: 25px;
+      }
+
+      .subtitle+p {
+        margin-top: 8px;
+      }
+    </mj-style>
+  </mj-head>
+  <mj-body>
+    {{> header }}
+
+    <mj-section padding="0px" text-align="center">
+      <mj-column width="100%" background-color="#fff" border-radius="10px">
+        <mj-text padding="10px 20px 10px 20px" font-size="18px" align="left" line-height="1.2">
+          <p>Hello,</p>
+
+          <p>You are getting this email because you either indicated an interest in participating in the {{ poolYear }} confidence pool or we thought you would be interested. All players are welcome, so even if you are not interested, please feel free to forward this along appropriately.</p>
+
+          <p>For those of you who have never done a confidence pool before, they are very easy to do. The objective is to
+            pick the winner of each game for the current week. When making your selections, you will assign a point value,
+            or "confidence" value, based on how sure you are of each pick. 1 means it could go either way and 16 means it's
+            a sure thing. If you are right, you win that amount of points and the player with the highest point total wins. Payouts are done to
+            the winner(s) each week and top 3 places at the end for the overall score. More general details about confidence pools
+            can be found here:</p>
+
+          <p><a href="http://ezinearticles.com/?How-to-Play-Fantasy-Football-Confidence-Pools&id=2596977">How to Play NFL Confidence Pools</a></p>
```

Comment:
@balexander82 Do we want to include a link to our support page here?  In addition or instead of this link I've used for 8 years now :)